articles/140_topic_and_service_name_mapping.md

```diff
@@ -198,51 +198,17 @@ Any topic or service name that contains any tokens (either namespaces or a topic
-That is acceptable because DDS partitions are allowed to contain forward slashes (`/`) unlike the DDS topics.
-
-This strategy also avoids DDS partitions from being exposed to the ROS API's.
-This is because the splitting of the ROS name into namespace and base name can occur inside the implementation of the ROS middleware interface, and so the ROS topic or service name will remain intact throughout the middleware agnostic code.
-This means that if someone chooses to implement ROS 2 on top of something other than DDS, then this strategy will not impact that implementation, as it applies only to DDS implementations.
-
-#### DDS Partitions Characteristics
-
-DDS partitions are implemented as an array of strings within the `DDS::Publisher` and `DDS::Subscriber` QoS settings.
-The array allows up to 64 items, and each string in the array is an individual partition.
-This means that every string of the array is an independently matchable entity when paired with the topic name and the other QoS settings.
-Put another way, a DDS DataWriter and DataReader will not connect to one another unless at least one of the partition strings in the array match, even if the topic names match.
-Sometimes special characters are used in the partition strings to express regular expressions matching (such as `+`, `*`, `^`), but they are not officially supported by the standard.
-The maximum size of each string in the array may depend on the DDS or RTPS implementation.
-You can read more about partitions in RTI's documentation:
-
-- [PARTITION_QosPolicy](https://community.rti.com/static/documentation/connext-dds/5.2.3/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_UsersManual/Content/UsersManual/PARTITION_QosPolicy.htm)
-
-The array of strings have no hierarchy and the order does not matter.
-Each entry in the partition array is directly combined with the DDS topic and they are not sequentially combined.
-If a publisher has two partition entries, e.g. `foo` and `bar` with a base name of `baz`, this would be equivalent to having two different publishers on these topics: `/foo/baz` and `/bar/baz`.
-Therefore we cannot use the array of strings to represent our implied namespace hierarchy and we have to continue to use the forward slash to accomplish this.
-
-#### Hierarchy with DDS Partitions
-
-Since DDS partitions have no hierarchy or order to the array of strings, this proposal would use only one of the strings in the array to hold the entire ROS name's namespace.
-Therefore we preserve the forward slashes (`/`) that exist in the ROS name's namespace already.
+Previously forward slashes (`/`) were disallowed in DDS topic names, now the restriction has been lifted and therefore the ROS topic names are first prefixed with ROS Specific Namespace prefix (described below) are then mapped directly into DDS topic names.  
```

Comment:
Can you reference any official statement or something indicating that it's now allowed?

Maybe the complete sentence can actually be part of the next section `ROS Specific Namespace Prefix`.

Reply:
Regarding the official statement is there any issue or thread that refers to this? I think this is a general statement which says that ROS topics are mapped directly to DDS topics and therefore is here.

Reply:
There is a DDS issue opened on this but hasn't been marked as "approved"/"resolved": https://issues.omg.org/issues/lists/dds-rtf5#issue-42236

Reply:
If that's the only pointer we have I'll go ahead and include it